documentation/example_codes/exampleParallelCall.m

```diff
@@ -0,0 +1,46 @@
+jd = JobDispatcher('JobName');              % Create job dispatcher
+
+[outputs, ...                               % Cell array of results from each worker
+ n_failed,...                               % Count of failed workers
+ task_ids,...                               % IDs of each worker (message folder?)
+ jd       ...                               % Object returns itself when done
+ ] = jd.start_job('exampleJobExecutor', ... % Name of class to run
+                  [1,2], ...                % Data passed to all classes via common_data
+                  10, ...                   % Number of "iterations" as int or cellarr of
+                  ...                       % structures of data to pass to each worker
+                  true, ...                 % Return outputs from workers
```

Comment:
It would be good to use full stops to mark the end of your comments, not in general but as you are running the comments over multiple lines and it will be easier to see where they end.
It would also be good to indent 2nd and subsequent lines of multiple line comments to make it clear where the continuation is,
It would be good to group alternatives and punctuate between them, so
Number of iterations as int, (comma to separate)
or cellarray of structures of data (maybe comma to separate)
to pass to each worker.